packages/lesswrong/lib/collections/tags/collection.ts

```diff
@@ -3,7 +3,7 @@ import { addUniversalFields, getDefaultResolvers, getDefaultMutations } from '..
 import { makeEditable } from '../../editor/make_editable'
 import { userCanCreateTags } from '../../betas';
 import { userIsAdmin } from '../../vulcan-users/permissions';
-import { schema } from './schema';
+import schema from './schema';
```

Comment:
This was the only collection that didn't use a default export, so I changed it to match the others.